batch_perf/results.txt

```diff
@@ -0,0 +1,35 @@
+On a 8 core 16MB - Apple MBP El Capitan
+
+                                                                             user     system      total        real
+redis input batch: 1000, lrange, ltrim, size: 5000, slow: 0ms            1.700000   0.070000   1.770000 (  0.362000)
```

Comment:
You executed this test with different input batch size and different slow parameters, I'm wondering if the different size parameter could affect also the result time? does this makes sense?